source/NVDAObjects/behaviors.py

```diff
@@ -261,6 +261,13 @@ def _getTextLines(self):
 		"""
 		return list(self.makeTextInfo(textInfos.POSITION_ALL).getTextInChunks(textInfos.UNIT_LINE))
 
+	def _reportNewLines(self, lines):
+		"""
+		Reports new lines of text using _reportNewText for each new line.

```

Comment:
Maybe state that subclasses can override this method to provide custom logic (such as filtering)?